README.md

```diff
@@ -279,3 +279,68 @@ return function MyForm() {
   );
 };
 ```
+
+### Using with `useActionState` Hook
+
+The action creators are compatible with React's `useActionState` hook, which provides a more flexible way to handle form state:
+
+```tsx
+"use client";
+import { useActionState } from "react-dom";
+
+// First, create a wrapper function for the action
+export async function signInAction(prevState: any, formData: FormData | null) {
+  return signIn(prevState, formData || new FormData());
```

Comment:
is this action supposed to be created by the `createFormAction` helper?
it seems totally unrelated to the lib
```suggestion
  return signIn(prevState, formData || new FormData());
```